File-Formats/XlsIO/Worksheet-Cells-Manipulation.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Worksheet Cells Manipulation | Syncfusion
+title: Worksheet Cells Manipulation in File Formats XlsIO Control | Syncfusion
 description: This section illustrates about cells manipulation in Excel worksheet using Syncfusion XlsIO (a .NET Excel library).
 platform: file-formats
 control: XlsIO
 documentation: UG
 keywords: c#, vb.net, excel, read excel, edit excel, edit excel cell, write excel cell, fill excel, write excel, update excel, syncfusion, xlsio
 ---
 
-# Worksheet Cells Manipulation
+# Worksheet Cells Manipulation in Fileformat PDF
```

Comment:
Change like this - Worksheet Cells Manipulation in File Formats XlsIo

Reply:
Resolved